### DISCUSSION TITLE
Hotkey to hide/show panel on the left? (regardless of which one it is)

### DISCUSSION THREAD

**Discussion by hi2u** [dbeaver/dbeaver#11700] [Q&A]
In vscode, I can hit `ctrl-b` to toggle show/hide of the panel on the left.  It doesn't matter which kind of panel is being displayed there, it just always works regardless.

Is it possible to bind a hotkey in dbeaver to do the same?

It would be very handy to be able to quickly toggle the panel to the left, without having to think about different keys depending on which panel it happens to be active at the time (or what is currently focused either).

**Comment by diorcula** [comment]
still looking for this one as well as there are way too many options

**Reply by diorcula** [reply]
also got met with unexpected program closing after assigning hotkey to show hide the "database navigator", so i had to reset the command
